proof/infoflow/Arch_IF.thy

```diff
@@ -2066,7 +2071,11 @@ lemma delete_objects_valid_ko_at_arm:
   apply(wp detype_valid_ko_at_arm do_machine_op_valid_ko_at_arm | simp add: ptr_range_def)+
   done
 
-
+lemma mask_neg_mask_is_zero:
+  "((x::obj_ref) && ~~ a) && a = 0"
```

Comment:
Would adding `word_bw_assocs` directly help? (in the simps where this this is used)

Reply:
(I haven't found the use site, so this might be left over, but there is a lot, so I might just have missed it)